domino/core/pom.xml

```diff
@@ -20,6 +20,12 @@
 				<extensions>true</extensions>
 				<configuration>
 					<singleton>true</singleton>
+					<manifestLocation>META-INF</manifestLocation>
```

Comment:
As far as I understand this setting, the MANIFEST.MF is put now into the /META-INF directory and no longer in the /target/.... directory